src/Analyzers/CSharp/Tests/PopulateSwitch/PopulateSwitchStatementTests.cs

```diff
@@ -1221,6 +1221,19 @@ public static void Test(MyEnum? myEnumValue)
 enum MyEnum
 {
     Value1, Value2
+}");
+        }
+
+        [Fact, WorkItem(61281, "https://github.com/dotnet/roslyn/issues/61281")]
+        public async Task TestNotInSwitchWithUnknownType()
+        {
+            await TestMissingInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        switch[||]
+    }
 }");
```

Comment:
can you add a test like: `var v = null switch[||]` thanks!

Reply:
This thing currently is parsed as a switch statement, so I've put it to the same test section